<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionOptionsSerialProperties</class>
 <widget class="QWidget" name="SessionOptionsSerialProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>341</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Port Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPortName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Baud Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBaudRate">
       <property name="maximum">
        <number>2147483647</number>
       </property>
       <property name="value">
        <number>115200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>DataBits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxDataBits">
       <item>
        <property name="text">
         <string notr="true">5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">8</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxParity">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>StopBits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxStopBits">
       <item>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxFlowControl">
       <property name="text">
        <string>RTS/CTS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxXEnable">
       <property name="text">
        <string>XON/XOFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
